### ISSUE TITLE
Replicate results from audio super resolution paper

### ISSUE THREAD

**Issue by achalshah20** [zayd/deep-audio-super-resolution#1]
Hi,

I think, you might have used this code to test DNN baseline in your and kuleshov's audio resolution paper. Can you tell me how to reproduce exact same results?

I am trying on VCTK corpus with r=2 but my SNR is stuck around 17. In your paper you achieved 19.9. So it will be helpful if you let me know what are the parameters needs to be changed to produce similar results

Best,
Achal

**Comment by Qpan77** [comment]
Hi,
     I see that you have reproduced the author's results, but the effect is not good enough. I want to ask you whether the training set inputs broadband voice or narrowband voice. Does the test set input narrowband voice?
